public/map-controller/map-controller.js

```diff
@@ -76,7 +78,10 @@ var MapController = P({
     }, layer);
     region_popup.setContent('<b>' + feature.properties.name + '</b>');
 
-    // Draw popup that tracks mouse movement.
+    // Store geo center for each region.
+    this.region_code_to_latlng[region_code] = layer.getBounds().getCenter();
```

Comment:
I think you can calculate it from the geojson without leaflet. Maybe just leave a note for now.